### PR TITLE
Create default build and unit test runners.

### PR DESCRIPTION
**Pull request by adrcunha**
Part of #101.

### CODE REVIEW

File: test/unit/test-helper.sh [hunk 1]
```diff
@@ -64,3 +64,10 @@ function mock_gcloud_function() {
   }
   "$@" 2>&1
 }
+
+# Convenience method to display a test failure and exit the script.
+# Parameters: $1 - message to display.
+function failed() {
```
Comment (chaodaiG) [inline test/unit/test-helper.sh]: this function name is also used as variable name in `scripts/presubmit-tests.sh`, is it on purpose?

File: test/unit/presubmit-custom-runners-tests.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/presubmit-integration-tests-common.sh
+
+function build_tests() {
+  RAN_BUILD_TESTS=1
```
Comment (chaodaiG) [inline test/unit/presubmit-custom-runners-tests.sh]: should it be `RUN` instead of `RAN`?

Comment (chaodaiG) [reply test/unit/presubmit-custom-runners-tests.sh]: oh never mind, I got it now

Comment (chaodaiG) [reply test/unit/presubmit-custom-runners-tests.sh]: just an idea, how about `BUILD_TESTS_RAN` ?

Comment (adrcunha) [reply test/unit/presubmit-custom-runners-tests.sh]: I think this is an unnecessary renaming.

File: test/unit/test-helper.sh (hunk 1 above)
Comment (adrcunha) [reply test/unit/test-helper.sh]: No, and one won't affect the other. Anyway, I'll rename for clarity.

**Comment by chaodaiG** [comment]
/lgtm
this is much easier to read

**Pull request merged**